presto-main/src/main/java/com/facebook/presto/sql/planner/plan/WindowNode.java

```diff
@@ -224,19 +225,26 @@ public boolean equals(Object obj)
         private final FrameBound.Type endType;
         private final Optional<Symbol> endValue;
 
+        private final Optional<Expression> originalStartValue;
```

Comment:
Can you please add a comment that is `This information is only used for printing the plan`.